ResumeParsing.md

```diff
@@ -32,13 +32,13 @@ Input Data
 
 Input should be provided in the POST request body, and content type should be either JSON or Form Vars (xml is not accepted)
 
-document: We accept .doc, .docx, .pdf, .rtf, .txt, .odt, .wps, and .pages documents given in a BASE64 encoded string.  Please note that .pages is not accepted by Textkernel; you will need to specify another parser. (Required)
+document: We accept .doc, .docx, .pdf, .rtf, .txt, .odt, .pages, and .wps documents given in a BASE64 encoded string.  Please note that .pages is not accepted by Textkernel; you will need to specify another parser. (Required)
 
```

Comment:
Seeing as language detection wont work (because preprocessing will always fail) should we be saying we still accept this format?